playbooks/roles/shadowsocks/vars/mirror.yml

```diff
@@ -19,14 +19,11 @@ shadowsocks_gui_url: "https://github.com/shadowsocks/shadowsocks-windows/release
 shadowsocks_gui_checksum: "sha256:4f932e61afb6bd1dd8b5c4c25c715f1623d3f574637d8154256531b4ef5000ac"
 
 # OS X
-# NOTE(@cpu): We're stuck using a beta here since only v1.5 supports AEAD
-#             cipher options. Once there is a finalized release we should drop
-#             the beta mirror
-shadowsocks_x_ng_version: "1.5-beta.2"
+shadowsocks_x_ng_version: "v1.6.1"
```

Comment:
I think you need to drop the `v` prefix in your `shadowsocks_x_ng_version`.It looks like that part of the file path is hardcoded in the `shadowsocks_x_ng_url` and so this mirror task is breaking in CI with a 404 trying to download a release from the `vv1.6.1` subdirectory:
```
 failed: [10.218.164.80] (item={u'url': u'https://github.com/shadowsocks/ShadowsocksX-NG/releases/download/vv1.6.1/ShadowsocksX-NG.v1.6.1.zip', u'checksum': u'sha256:b4fbacdbade6d2964c309fb630972973a1c3ac5aafb94d6a7c06fa879840886e'}) => {"dest": "/var/www/streisand/mirror/shadowsocks", "failed": true, "gid": 33, "group": "www-data", "item": {"checksum": "sha256:b4fbacdbade6d2964c309fb630972973a1c3ac5aafb94d6a7c06fa879840886e", "url": "https://github.com/shadowsocks/ShadowsocksX-NG/releases/download/vv1.6.1/ShadowsocksX-NG.v1.6.1.zip"}, "mode": "0755", "msg": "Request failed", "owner": "www-data", "response": "HTTP Error 404: Not Found", "size": 4096, "state": "directory", "status_code": 404, "uid": 33, "url": "https://github.com/shadowsocks/ShadowsocksX-NG/releases/download/vv1.6.1/ShadowsocksX-NG.v1.6.1.zip"}
```